File: tests/toaster/LOCAL+C2Jno/jt.js
let count = 0;



jtask* function you(str) {
    assert(12==12);
    count++;
    console.log("Message received: ", str, " local count ", count);
    coverage();
}

setInterval(()=> {
    coverage();
    console.log("hello.. main loop ");
}, 1000);

File: tests/toaster/LOCAL+C2Jno/jt.c
void you(char *s);

jtask* localme(int c, char *s) {
    while(1) {
        jsleep(2000000);
        printf("############-->>> Hello  ME  %d... %s\n", c, s);
    }
}

jtask* localyou(int c, char *s) {
    while(1) {
        jsleep(1000000);
        printf("############-->>> Hello YOU  %d, %s\n", c, s);
        you(s);
    }
}

int main(int argc, char *argv[])
{
    assert(12==13);
    localme(10, "cxxxxyyyy");
    localyou(10, "a-message-for-j");
    return 0;
}
